addons/mail/static/src/core/common/plugin/mail_composer_plugin.js

```diff
@@ -54,4 +57,13 @@ export class MailComposerPlugin extends Plugin {
             this.config.composerPluginDependencies.onFocusout
         );
     }
+    handlePasteHtmlOverride(html) {
+        if (this.config.composerPluginDependencies.isInChatter) {
+            return false;
+        }
+        // markup: escape html from clipboard is considered safe as it has been sanitized
+        this.dependencies.dom.insert(htmlToTextContentInline(markup(html)));
```

Comment:
This is a bad markup pattern, I don't see where it would be sanitized in the current diff.
The markup should be moved on the line where it is actually sanitized.